README.md

```diff
@@ -2,17 +2,14 @@
 
 # Project Name
 
-> One paragraph statement about the project.
+> Portfolio Project.
 
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+This project is been done to test my knowledge on Flexbox and semantic tags used in HTML by Building the header Section of a Mobile version portfio website where aligning of elements and the use of figma to replicant designs with code was very vital
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML,CSS,FLEXBOX,CSSGRID & JAVASCRIPT
+- Git & Github
 
 ## Live Demo
```

Comment:
[OPTIONAL ] Your live demo link is currently broken. You can consider removing it for now because its not mandatory for this project.

Reply:
thank you very much. I have followed your suggestion and removed the Live Demo link.